server/autolinkplugin/config.go

```diff
@@ -78,6 +83,35 @@ func (p *Plugin) SaveLinks(links []autolink.Autolink) error {
 	return nil
 }
 
+// GetPreConFigLinks gets the preconfigured links from the plugin config
```

Comment:
What are your thoughts on moving this function to /autolinkplugin/preconfigured_links.go?